gradio/components/line_plot.py

```diff
@@ -172,6 +176,11 @@ def __init__(
             every=every,
             inputs=inputs,
         )
+        if interactive is not None:
+            warnings.warn(
+                "The `interactive` parameter is deprecated and will be removed in a future version. "
+                "The LinePlot component is always interactive."
```

Comment:
You mean its always static, right? We use "interactive" to mean that a user can change the value of the component by interacting with it via the Gradio UI

Reply:
Yeah I suppose. Before this PR, `interactive` for plots specifically meant that you could hover over point for extra info, it had nothing to do with the normal static vs interactive definition.